src/components/CardComponent.jsx

```diff
@@ -1,20 +1,18 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { Card } from 'semantic-ui-react';
 import PropTypes from 'prop-types';
 
-class CardComponent extends Component {
+class CardComponent extends React.Component {

```

Comment:
Why the change to this line? It is unnecessary.  Revert the import to `import React, { Component } from 'react';` and the class definition to `class CardComponent extends Component`